### PR TITLE
Allow temporary opt-in to core

### PR DESCRIPTION
**Pull request by sndrs**
enable per-view core opt-in with `#core`

### CODE REVIEW

File: common/app/templates/headerInlineJS/shouldEnhance.scala.js [hunk 1]
```diff
@@ -31,7 +32,8 @@
     };
 
     window.shouldEnhance = !personPrefersCore() && !isOlderDevice();
-})(navigator, window)
+    window.shouldEnhance || console && console.info && console.info("THIS IS CORE");
```
Comment (davidrapson) [inline common/app/templates/headerInlineJS/shouldEnhance.scala.js]: :smile:

**Comment by davidrapson** [comment]
:+1:

**Comment by stephanfowler** [comment]
:+1: Total Game Change

File: common/app/templates/headerInlineJS/shouldEnhance.scala.js (hunk 1 above)
Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/shouldEnhance.scala.js]: If `window.console` is undefined, referencing `console` alone will lead to a `ReferenceError`, so you might want to change this to `window.console && window.console.info && …`

We do this enough in pre-module-loader JS to warrant a simple abstraction for it (i.e. `hasConsole` fn)

**Pull request merged**